mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -18,10 +28,25 @@ AUI.add(
 
 				prototype: {
 					initializer: function(config) {
+						new A.CharCounter(
+							{
+								input: '.input-item',
+
+								counter: '.counter',
+
+								maxLength: 35
+							});
```

Comment:
Fix spacing and indentation. See any object argument in portal or plugins repositories.